### ISSUE TITLE
right joystick speed

### ISSUE THREAD

**Issue by vdegenne** [Tylemagne/Gopher#116]
Before asking my question I would want to thanks the beautiful people who took time and effort to build this amazing program. I suffered from hands aches over the past years from over using my desk (I am developer myself) and since I have a xbox controller and found this project the pain has been gone. This program is not just a tool, it's also a medication and maybe a step forward in working ergonomic evolution ?

Anyways the only unfavorable point I found is that the right joystick is too sensitive.
Is there any way to reduce the speed. For instance If pressing left and right bumpers toggle the speed of the left joystick.
Does someone know (hopefully one of the dev) how to achieve the same but for the right one ?
Scrolling pages on Chrome is really difficult because as soon as I just move a little bit the joystick to the bottom the scroll goes crazy and scroll something like 2 paragraphs down, I have to tweak the joystick to reposition myself on the right spot (waste of time).

Thanks a lot for the help

**Comment by benhall847** [comment]
I'm also having the same issue, the only thing I would like to do is change the sensitivity and dead-zone of the joysticks.

**Comment by vdegenne** [comment]
@benhall847 Unfortunately it seems that the maintainers are long time gone. Hope someone forks this and make the changes. I'd do it but I am really not mastering C++

**Comment by mike1084** [comment]
First of all, this was truly a heart-warming post, and something that inspires me to get back into working on this again! I am not part of the devs or anything, but as this was open source, when I picked it up I walked into many of the issues that people face. Remapping buttons, switching sticks, all things that would make it slightly better.
Ahem, to get to the point; I made some decent work in improving the code, part of which I posted in an issue regarding stick movement in games. Next to that, there have been a bunch of commits to this Github but they were simply never folded into a new official release. All I could say is pull the whole Github including commits, and build it yourself. But that depends on what kind of developer you are, of course. ;)
I'm currently rather preoccupied, but should have more free time on my hands in roughly 2-3 months. Once I do, I'll be sure to get into this again, see where the commits went, and perhaps make a fork so that you all could at least have the most up to date version as an official release, possibly paving the way for future development. 

TL;DR good to hear that this software is helping you (and many others), and I'd love to see if we could make this even better!

**Comment by achimmihca** [comment]
To my surprise, parameters SCROLL_SPEED, DEAD_ZONE, and some more have already been implemented. However, the last release (Gopher360 0.989) did not include them.

I added a PR (#143) to add these parameters to the default config.
You might want to try the more recent [build](https://github.com/achimmihca/Gopher360/releases/tag/v1.0-achimmihca-fork) I just created that contains these changes.

**Comment by vdegenne** [comment]
@achimmihca Thanks a lot! You are amazing!

**Comment by achimmihca** [comment]
> @achimmihca Thanks a lot! You are amazing!

Many thanks!
Recently, I started to use [GitHub Sponsors](https://github.com/sponsors/achimmihca). If you want, you can buy me a coffee by selecting a one-time tier.
